docs/preview/connected-registry/quickstart-deploy-connected-registry-iot-edge-cli.md

```diff
@@ -25,7 +25,21 @@ In this quickstart, you use [Azure Container Registry][container-registry-intro]
 
 ## Before you begin
 
-This tutorial requires an Azure IoT Edge device to be set up upfront. You can use the [Deploy your first IoT Edge module to a virtual Linux device](../iot-edge/quickstart-linux.md) quickstart guide to learn how to deploy a virtual IoT Edge device. The connected registry is deployed as a module on the IoT Edge device.
+This tutorial requires an Azure IoT Edge device to be set up upfront. You can use the [Deploy your first IoT Edge module to a virtual Linux device](../iot-edge/quickstart-linux.md) quickstart guide to learn how to deploy a virtual IoT Edge device. The connected registry is deployed as a module on the IoT Edge device. 
+
+To install the latest 1.2 version of iotedge agent, login to the IoT device, open `/etc/iotedge/config.yaml`, search the section for `edgeAgent`, update the image version to 1.2.0 as the following.
```

Comment:
is the instruction below for existing device with old version before 1.2? does new device by default install 1.2?

Reply:
The instruction let you deploy the vm from an existing template. I just tried it and it used 1.1 version.